### DISCUSSION TITLE
[Bug report] The Post Prefix functionality missing from 0.46.1

### DISCUSSION THREAD

**Discussion by bjazmoore** [GetPublii/Publii#1653] [Bug Report] [answered]
### Operating system

Windows 11

### Publii version

0.46.1

### Issue type

Application

### Bug description

I did not check this in version 0.46.0, but in version 0.46.1 I cannot find the Post Prefix functionality that is discussed in the release blog post and pictured here (from the blog post):

![shot_240831_212747](https://github.com/user-attachments/assets/76e4572d-dcd0-4113-9ba9-488d0cf9dc1f)
 
I see the following on my system:

![shot_240831_212723](https://github.com/user-attachments/assets/414149e8-7518-4746-b5b3-1cbd0516beb5)

I have tried checking with a brand new web just in case something I did on my existing page cause an issue.

### Steps to reproduce

Install Publii 0.46.1 -> create a web -> go to Site settings -> go to URLS.

**Comment by bobmitro** [comment]
This is because the Post Prefix requires enabling Pretty URLs. If the URL structure isn’t directory-based (i.e., Pretty URLs are disabled), setting the blog directory or any other directory won't be possible. So, to use this feature, you’ll need to enable Pretty URLs first, and then you can set up the blog directory.

**Reply by bjazmoore** [reply]
I wish the Discussion system would let me mark multiple answers.  This is the answer for sure, and I thank you so much Bob.  I added some commentary on the process below and marked it as the answer.  I hope that is ok.

**Comment by bjazmoore** [comment, marked as answer]
Thank you Bob.  I missed that!

So even after the pointer I struggled a little bit with this one.  For anyone else looking to add a Blog page after setting a specific page as the home page here are the steps to take:

1) Enable Pretty URL's (Site settings -> URLs)
2) Set a page you have previously created as your new home page (Site settings -> SEO -> Homepage section)
3) Set a new Posts prefix - this can be anything you want - I think "blog" makes sense though. (Site settings -> URLS)
4) Don't forget to add a couple posts so you can see this in action.  You might also want to edit the Hero section for the blog page (Theme -> Layout -> Hero section)
5) You will not be able to access your new blog unless you add a link to it.  Create a menu and add a link to the "Posts index link".  Here is what my test page looks like:

![shot_240902_110924](https://github.com/user-attachments/assets/78858748-88c2-4aa7-b606-a147eabc8bc0)

Ok - now test it out!  Magic!  And some really impressive magic at that.

Thanks so much to the Publii team who made this possible.
